.github/actions/run-workspace-tests/action.yml

```diff
@@ -57,10 +57,17 @@ runs:
         cache-to: type=gha,mode=max
         build-args: BASE_IMAGE=${{ steps.ecr-login.outputs.registry }}/${{ inputs.ugbio_base_image }}
 
-    - name: Run tests in the container
+    - name: Run python (if exist) tests in the container
+      working-directory: src/${{ inputs.workspace-folder }}
       shell: bash
       run: |
-        docker run --rm -v ${{ github.workspace }}/src:/workdir/src ${{ inputs.workspace-folder }} pytest /workdir/src/${{ inputs.workspace-folder }}
+        if [ -f pyproject.toml ]; then
```

Comment:
Why do we need pyproject.toml to run tets? I didn't understand the connection between the two

Reply:
Can you suggest another way to check if this is a python project that we should run pytest on?

Reply:
just run pytest. It should run all the tests in python files it can find.